test/mocha.opts

```diff
@@ -1 +1 @@
---timeout 2000
+--timeout 10000
```

Comment:
Will timeout when testing `SendMessageBatchError` as we retry `10` times with the exponential backoff function

Reply:
revert and fix tests to run fast